blog/_posts/2018-11-18-RGSoCTeamsGOTOBerlin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: RGSoC Teams at GOTO Berlin!
+layout: post
+created_at: Tue Dec 18 2018
+permalink: blog/2018-11-18-RGSoCTeamsGOTOBerlin
+current: blog
+author: Avneet Kaur
+twitter: TeamSectumsem18
+categories:
+ - student-posts
+---
+
+## Our experiences at GOTO Berlin 
+
+### Arati- Team Popcorn  
+
+The GoTo Conference, my first conference, concluded an already
+unforgettable summer and I have RGSOC to thank for both. From the morning
+train ride mentally preparing for the day, to the evening capped off by a social
+event or a chat with a new acquaintance, I was constantly exposed to new
+concepts, people, tools, companies and more.
+For instance, “Functional Programming in 40 Minutes“ by Russ Olsen made
+me curious about learning Clojure, particularly since he said learning languages
+was a hobby of his. The timing of the conference couldn’t have been better. It
+came right at the point where I was starting to try to figure out how to apply my
+math background to programming. I had the opportunity to meet several people
+involved in machine learning, a field that is constantly brought up when I mention
+my intention, and attended enough talks about the subject to have an
+understanding of what it involves.
+The main impression I was left with, however, is that of being intensely
+inspired. I had expected to be exposed to new concepts, technical information
+and tools. What I hadn’t expected was to gain insights in other areas of my life.
+One particularly inspiring talk was “How Software Can be a Force for Good”.
+Hearing Francesco Mondora describe the unconventional approach to business
+at his company renewed my hope in the impact a single person can have on the
+world. Their emphasis on making a positive difference, whether it is to the
+environment or in the lives of the people working at the company, has given me a
+lot to think about for my own life. It has shown me the work place as a context to
+not only live out your values fully, but also to support them in a way that only a
+team can.
+
+### Rupal and Avneet- Team Sectumsempra
+My teammate Rupal and I, Avneet had the opportunity to attend GOTOBerlin 2018, all thanks to RGSoC organizers and GOTOBerlin organizers as well. 
+It was a 3 day conference with diversified tracks. We were particularly interested in the Applied AI and Blockchains track of the conference and mainly attended the talks in that domain. It was an experience of a lifetime and we learnt so much during the 3 days at the conference. So here’s an insight into how our days at the conference were spent. 
+
+![Alt Text](/img/blog/2018/gotoBerlinTeamSectumsempra1.jpg)
+<div class="image-credits">Avneet (Left), Djana Gunther (conference organizer) (Centre), Rupal (Right) (source: Avneet's phone camera - Moto G4 Plus)</div>
```

Comment:
small typo: Dajana (instead of “Djana”). I think the last name also has an ü (i.e. “Günther”) but I'm happy to add that myself if needed!
Also, as a source just the name of the person the photo originated from is enough (i.e. “Avneet Kaur”).